cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -614,6 +614,42 @@ def _validate(self, name: str):
             self._add_failure(f"It is forbidden to use '{name}' as a name.", FailureLevel.ERROR)
 
 
+class SharedStorageMountDirValidator(Validator):
+    """
+    Shared storage mount directory validator.
+
+    Make sure the mount directory is not the same as any reserved directory.
+    """
+
+    def _validate(self, mount_dir: str):
+        reserved_directories = [
+            "/bin",
```

Comment:
Could you put them in alphabetical order? it would be easier to understand if we're missing something

Reply:
Done